demos/common/processing/requests_pool.h

```diff
@@ -0,0 +1,63 @@
+/*
+// Copyright (C) 2018-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+#pragma once
+
+#include <map>
+#include <atomic>
+#include <samples/ocv_common.hpp>
+
+/// This is class storing requests pool for asynchronous pipeline
+/// 
+class RequestsPool
+{
+public:
+    RequestsPool(){}
```

Comment:
Is this constructor needed? It doesn't seem to be used anywhere.

Reply:
I assume this is also applicable for `ImageInputData() {}`.

Reply:
Ping

Reply:
Pong. This constructor is deleted, ImageInputData default constructor should be left as is (as it might might be useful later)

Reply:
I'm sick of repeating that you never know what might be useful later. Whatever.